src/Raven.Server/Documents/Replication/ResolveConflictOnReplicationConfigurationChange.cs

```diff
@@ -105,56 +128,29 @@ public void RunConflictResolversOnce()
                     _log.Info("Failed to run automatic conflict resolution", e);
             }
         }
-
-
-        private class ResolveConflictsCommand : TransactionOperationsMerger.MergedTransactionCommand
+        
+        private class PutResolvedConflictsCommand : TransactionOperationsMerger.MergedTransactionCommand
         {
-            private readonly List<DocumentConflict> _conflicts;
+            private readonly List<DocumentConflict> _resolvedConflicts;
             private readonly ResolveConflictOnReplicationConfigurationChange _resolver;
 
-            public ResolveConflictsCommand(List<DocumentConflict> conflicts, ResolveConflictOnReplicationConfigurationChange resolver)
+            public PutResolvedConflictsCommand(List<DocumentConflict> resolvedConflicts, ResolveConflictOnReplicationConfigurationChange resolver)
```

Comment:
What kind of concurrency errors should we expect here? Underneath, `PutResolvedDocument` just do the PUT with `null` as the expected etag? I guess it shouldn't?